recipes/blast/build.sh

```diff
@@ -68,31 +63,32 @@ export AR="${AR} rcs"
     --without-vdb \
     --with-z=$PREFIX \
     --with-bz2=$PREFIX \
-    --with-boost=$PREFIX \
-    --without-gcrypt \
-    --with-nettle=$PREFIX \
     --with-z=$PREFIX \
     --without-krb5 \
     --without-openssl \
-    --with-gnutls=$PREFIX
+    --without-gnutls \
+    --without-gcrypt
 
-projects="algo/blast/ app/ objmgr/ objtools/align_format/ objtools/blast/"
+apps="blastp.exe blastn.exe blastx.exe tblastn.exe tblastx.exe psiblast.exe rpsblast.exe rpstblastn.exe"
+apps="$apps makembindex.exe segmasker.exe dustmasker.exe windowmasker.exe deltablast.exe"
+apps="$apps makeblastdb.exe blastdbcmd.exe blastdb_aliastool.exe convert2blastmask.exe blastdbcheck.exe"
+apps="$apps makeprofiledb.exe blast_formatter.exe"
```

Comment:
Long list. Sounds ok to me. What is the benefit of giving apps vs giving projects? I assume you wanted to add something?

Reply:
Rewrap to 77 chars per line? Shouldn't make much of a difference, but easier to read here or via email.

Reply:
> What is the benefit of giving apps vs giving projects?
This is more specific, it builds only those applications.

> Rewrap to 77 chars per line? 
Will do.

Reply:
thanks